<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Second_Screen</class>
 <widget class="QMainWindow" name="Second_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>775</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>817</width>
    <height>775</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>751</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/5655049.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>811</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="repeat">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#repeat{
	background-color: rgba(0,0, 0, 0);
	color: rgba(85, 98, 112, 255);
}
QPushButton#repeat:hover{
	color: rgba(131, 96, 53, 255);
}
QPushButton#repeat:pressed{
	padding-left: 5px;
	padding-top: 5px;
	color: rgba(91, 88, 53, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Images/repeatbtn.png</normaloff>../Images/repeatbtn.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="preButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>45</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#preButton{
	background-color: rgba(0,0, 0, 0);
	color: rgba(85, 98, 112, 255);
}
QPushButton#preButton:hover{
	color: rgba(131, 96, 53, 255);
}
QPushButton#preButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	color: rgba(91, 88, 53, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Images/previous.png</normaloff>../Images/previous.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playButton1">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>35</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#playButton1{
	background-color: rgba(0,0, 0, 0);
	color: rgba(85, 98, 112, 255);
}
QPushButton#playButton1:hover{
	color: rgba(131, 96, 53, 255);
}
QPushButton#playButton1:pressed{
	padding-left: 5px;
	padding-top: 5px;
	color: rgba(91, 88, 53, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Images/play.png</normaloff>../Images/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>35</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pauseButton{
	background-color: rgba(0,0, 0, 0);
	color: rgba(85, 98, 112, 255);
}
QPushButton#pauseButton:hover{
	color: rgba(131, 96, 53, 255);
}
QPushButton#pauseButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	color: rgba(91, 88, 53, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Images/pausebtn.png</normaloff>../Images/pausebtn.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="font">
        <font>
         <pointsize>45</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#nextButton{
	background-color: rgba(0,0, 0, 0);
	color: rgba(85, 98, 112, 255);
}
QPushButton#nextButton:hover{
	color: rgba(131, 96, 53, 255);
}
QPushButton#nextButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	color: rgba(91, 88, 53, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Images/next.png</normaloff>../Images/next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shuffleButton">
       <property name="font">
        <font>
         <pointsize>39</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#shuffleButton{
	background-color: rgba(0,0, 0, 0);
	color: rgba(85, 98, 112, 255);
}
QPushButton#shuffleButton:hover{
	color: rgba(131, 96, 53, 255);
}
QPushButton#shuffleButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	color: rgba(91, 88, 53, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Images/shuffle.png</normaloff>../Images/shuffle.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backButton1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#backButton1{
background-color:rgba(255,107,107,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#backButton1:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(255,84,84,200);
background-position:calc(100%-10px)center;
}
QPushButton#backButton1:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(255, 107, 107, 255), stop:1 rgba(244, 168, 168, 219));
}
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>761</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="timebox">
     <item>
      <widget class="QLabel" name="start_time">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider">
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal {
	border-radius: 1px solid;
	height: 10px;
	background-color: rgb(153, 153, 153);
}
QSlider::handle:horizontal {
	background-color: rgb(255, 255, 255);
	width: 10px;
	margin: -1px -1px;
	border-radius: 5px;
	border: 1px solid rgb(85, 85, 255);
}
QSlider::add-page:horizontal {
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgba(188, 231, 215, 255), stop:1 rgba(122, 215, 255, 255));
}
QSlider::sub-page:horizontal {
	background-color: qlineargradient(x1:1, y1:0.585, x2:0, y2:0.556818, stop:0 rgba(132, 76, 171, 255), stop:1 rgba(122, 130, 255, 255));
}
QSlider::handle:horizontal:hover {
	background-color: rgb(112, 128, 144);
	border-color: rgb(112, 128, 144);
}</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="end_time">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="PlayListImage">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>330</width>
      <height>330</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>330</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/listimg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentSong">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="musicPlaying">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>570</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
	border-radius: 1px solid;
	height: 2px;
	background-color: rgb(42, 42, 42);
}
QSlider::handle:horizontal {
	background-color: rgb(255, 255, 255);
	width: 10px;
	margin: -5px -1px;
	border-radius: 5px;
	border: 1px solid rgb(42, 42, 42);
}QSlider::add-page:horizontal {
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgba(188, 231, 215, 255), stop:1 rgba(122, 215, 255, 255));
}
QSlider::sub-page:horizontal {
	background-color: qlineargradient(x1:1, y1:0.585, x2:0, y2:0.556818, stop:0 rgba(132, 76, 171, 255), stop:1 rgba(122, 130, 255, 255));
}
QSlider::handle:horizontal:hover {
	background-color: rgb(112, 128, 144);
	border-color: rgb(112, 128, 144);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/volumel.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
